tsconfig.json

```diff
@@ -1,10 +1,7 @@
 {
   "extends": "aegir/src/config/tsconfig.aegir.json",
   "compilerOptions": {
-      "outDir": "dist"
```

Comment:
Can you please revert all the whitespace changes to this file?

Reply:
At the moment, the CI is failing due to some types.

```
> aegir build --no-bundle

[11:21:13] Clean ./dist [started]
[11:21:13] Clean ./dist [completed]
[11:21:13] Generate types [started]
src/http.js(140,7): error TS2345: Argument of type 'AbortController | AbortController' is not assignable to parameter of type 'AbortController'.
  Type 'import("/home/runner/work/js-ipfs-utils/js-ipfs-utils/node_modules/abort-controller/dist/abort-controller").default' is not assignable to type 'AbortController'.
    Types of property 'signal' are incompatible.
      Property 'abort' is missing in type 'import("/home/runner/work/js-ipfs-utils/js-ipfs-utils/node_modules/abort-controller/dist/abort-controller").AbortSignal' but required in type 'AbortSignal'.
[11:21:16] Generate types [failed]
[11:21:16] → Command failed with exit code 1: tsc --build /home/runner/work/js-ipfs-utils/js-ipfs-utils/tsconfig-types.aegir.json
Command failed with exit code 1: tsc --build /home/runner/work/js-ipfs-utils/js-ipfs-utils/tsconfig-types.aegir.json
```

This was a quick and dirty solution to fix in order to test in codesandbox but without success. I was targeting this PR instead and the sandbox did not like it either.